Plugins/RawCMS.Plugins.Core/Configuration/AuthConfig.cs

```diff
@@ -15,34 +15,44 @@ namespace RawCMS.Plugins.Core.Configuration
 {
     public enum OAuthMode
     {
-        Standalone,
-        External
+        JWT,
+        Introspection
     }
 
-    public class AuthConfig
+    public class ExternalProvider
     {
-        public AuthConfig()
-        {
-            Mode = OAuthMode.Standalone;
-            Authority = "http://localhost:50093";
-            ClientId = "raw.client";
-            ClientSecret = "raw.secret";
-            ApiResource = "rawcms";
-        }
-
         public OAuthMode Mode { get; set; }
+        public string SchemaName { get; set; }
```

Comment:
@keggyy  All this params is on the same level. To allow extensibility the configuration should be a Dictionary<plugin name, JObject> where all implementation has its config.